packages/web-components/src/components/ic-toggle-button-group/ic-toggle-button-group.stories.mdx

```diff
@@ -0,0 +1,447 @@
+import {
+  Meta,
+  Story,
+  Canvas,
+  ArgsTable,
+  Description,
+} from "@storybook/addon-docs";
+import { html } from "lit-html";
+import readme from "./readme.md";
+
+<Meta
+  title="Web Components/Toggle Button Group"
+  component="Toggle Button Group"
+/>
+
+<Description markdown={readme} />
+
+### Select type
+
+<Canvas>
+  <Story name="Select type" parameters={{ loki: { skip: true } }}>
+    {(args) =>
+      html`
+        <span>Single and manual</span>
+        <ic-toggle-button-group
+          select-type="single"
+          label="Label for single and manual"
+          accessible-label="accessible label for the toggle button group"
```

Comment:
Sorry to keep adding comments like this - but if you could replace "accessible label for the toggle button group" with "single select toggle group" it would be so much more useful

Same with the other two groups, you could have "auto select toggle group" and "multiple select toggle group"

Reply:
This comment goes for all the groups in both web component and react stories. 
Thanks :)